sample_app/config/routes.rb

```diff
@@ -0,0 +1,13 @@
+Rails.application.routes.draw do
+  
+  root 'static_pages#home'
+
+  get 'static_pages/home'
+  get 'static_pages/help'
+  get 'static_pages/about'
+  get 'static_pages/contact'
```

Comment:
This line works.  But it makes the contact page available at "http://host/static_pages/contact".  Is that what you want?  Or do you want the page to be at "http://host/contact" ?

(similar for all the other ones above)